### PR TITLE
F/wpc 524 color picker peered articles color

### PR DESCRIPTION
**Pull request by camilleory**
I replaced the theme color in edit peer info by a color picker. 

I also replaced the color field in peer by a div showing the color (instead of the hex number).

image
image

### CODE REVIEW

File: packages/editor/src/client/panel/peerEditPanel.tsx [hunk 1]
```diff
@@ -220,7 +220,13 @@ export function PeerEditPanel({id, hostURL, onClose, onSave}: PeerEditPanelProps
                 {profile?.name}
               </DescriptionListItem>
               <DescriptionListItem label={t('peerList.panels.themeColor')}>
-                {profile?.themeColor}
+                <div
```
Comment (roosnic1) [inline packages/editor/src/client/panel/peerEditPanel.tsx:223]: I recommend to still display the hex code of the theme color.

**Review by roosnic1 — CHANGES_REQUESTED** [review]
Looks good but keep the hex color code.

**Review by penk85 — DISMISSED** [review]
:+1:  very nice

File: packages/editor/src/client/panel/peerEditPanel.tsx (hunk 1 above)
Comment (camilleory) [reply packages/editor/src/client/panel/peerEditPanel.tsx:223]: Ok I'll change it back, can you tell me why it's better with the hex code ?

**Review by penk85 — DISMISSED** [review]

**Review by lakeside763 — DISMISSED** [review]
nice one camille

File: packages/editor/src/client/panel/peerEditPanel.tsx [hunk 2]
```diff
@@ -220,7 +220,17 @@ export function PeerEditPanel({id, hostURL, onClose, onSave}: PeerEditPanelProps
...
+                  <div
+                    style={{
+                      backgroundColor: profile?.themeColor,
+                      width: '30px',
+                      height: '20px',
+                      padding: '5px',
+                      marginLeft: '5px'
+                    }}></div>
+                </div>
               </DescriptionListItem>
```
Comment (camilleory) [inline packages/editor/src/client/panel/peerEditPanel.tsx:234]: Is this way of adding css a good way in this case ?

Comment (roosnic1) [reply packages/editor/src/client/panel/peerEditPanel.tsx:234]: for a few css properties yes.

**Review by roosnic1 — APPROVED** [review]
👍🏾

**Review by bishoymelek — APPROVED** [review]
🎉🎉

**Pull request merged**